pkg/graph/schema.resolvers.go

```diff
@@ -423,36 +460,80 @@ func (r *systemIntakeResolver) EaCollaboratorName(ctx context.Context, obj *mode
 	return obj.EACollaboratorName.Ptr(), nil
 }
 
-func (r *systemIntakeResolver) EaSupportRequest(ctx context.Context, obj *models.SystemIntake) (*bool, error) {
-	return obj.EASupportRequest.Ptr(), nil
-}
-
 func (r *systemIntakeResolver) EuaUserID(ctx context.Context, obj *models.SystemIntake) (string, error) {
 	return obj.EUAUserID.String, nil
 }
 
-func (r *systemIntakeResolver) ExistingContract(ctx context.Context, obj *models.SystemIntake) (*string, error) {
-	return obj.ExistingContract.Ptr(), nil
+func (r *systemIntakeResolver) FundingSource(ctx context.Context, obj *models.SystemIntake) (*model.SystemIntakeFundingSource, error) {
+	return &model.SystemIntakeFundingSource{
+		IsFunded:      obj.ExistingFunding.Ptr(),
+		FundingNumber: obj.FundingNumber.Ptr(),
+		Source:        obj.FundingSource.Ptr(),
+	}, nil
 }
 
-func (r *systemIntakeResolver) ExistingFunding(ctx context.Context, obj *models.SystemIntake) (*bool, error) {
-	return obj.ExistingFunding.Ptr(), nil
-}
+func (r *systemIntakeResolver) GovernanceTeams(ctx context.Context, obj *models.SystemIntake) (*model.SystemIntakeGovernanceTeam, error) {
```

Comment:
This is the one data transformation that I think might still really belong to the front end, but I think putting it in the resolver is a good first step toward getting all our endpoints using graph queries. Then, we should figure out how to refactor this.